app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,35 @@
+  <h1 class= "section" >Expenses</h1>
+
+  <div class= "expenses-container">
```

Comment:
- [ ] Kindly add a back button so when the user clicks on the "Back" button (<), the user navigates to the home page, currently there is no way to go back to the homepage on your app from the expenses page, adding this will create a better user experience 👍🏿